doc/cygnus-ngsi-ld/installation_and_administration_guide/introduction.md

```diff
@@ -29,13 +25,22 @@ If your aim is to create a new sink for cygnus-ngsi-ld, or expand it in some way
 ## Structure of the document
 Apart from this introduction, this Installation and Administration Guide mainly contains sections about installing, configuring, running and testing cygnus-ngsi-ld. The FIWARE user will also find useful information regarding multitenancy or performance tips. In addition, sanity check procedures (useful to know wether the installation was successful or not) and diagnosis procedures (a set of tips aiming to help when an issue arises) are provided as well.
 
-It is very important to note that, for those topics not covered by this documentation, the related section in cygnus-common applies. Specifically:
 
-* Hardware requirements.
-* Flume environment configuration.
-* log4j configuration.
-* Running as a process.
-* Management interface.
-* Logs and alarms.
+* [Introduction](./introduction.md)
```

Comment:
This is a weird self-linking of the same document... I'd suggest to remove this entry.

Reply:
Done